sourdough/sourdough.py

```diff
@@ -380,6 +382,44 @@ def detectVSphereHost():
     except socket.error:
       this.logger.info('Cannot connect to vSphere hypervisor %s', hostname)
 
+def connectVcenter():
+  '''
+  Connect to Vcenter Server
+
+  Returns the UUID and Vcenter Connect objects
+  '''
+  uuid = getUUID()
+  loadSharedLogger()
+  secure=ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+  secure.verify_mode=ssl.CERT_NONE
+
+  this.logger.debug('secure.verify_mode=ssl.CERT_NONE')
+
+  vSphereSettings = loadVSphereSettings()
+  if not vSphereSettings:
+    this.logger.error('Could not load vSphereSettings!')
+    return None
+  this.logger.debug('vSphereSettings: %r', vSphereSettings)
+
+  try:
+    hostname = vSphereSettings['hostname']
```

Comment:
Move these checks into loadVSphereSettings and raise the error there if it can't load all the required keys

Reply:
Please check 9c078c7a10f23335cec3283d95dd8d8b76f04921